engine/src/force_feedback.cpp

```diff
@@ -205,12 +205,12 @@ void ForceFeedback::stopEffect( unsigned int eff_nr )
 
 void ForceFeedback::init()
 {
-    if (!game_options()->force_feedback) {
+    if (!configuration().joystick.force_feedback) {
         VS_LOG(info, "force feedback disabled in config file");
         return;
     }
     char devname[200];
-    sprintf( devname, "/dev/input/event%d", game_options()->ff_device );
+    snprintf( devname, 199, "/dev/input/event%d", configuration().joystick.ff_device );
```

Comment:
should the `199` be a configuration option?

Reply:
Maybe it should be a constant, though, instead of a "magic number." Like `BUFFER_SIZE` or something.

Reply:
Changed to a constexpr named `BUFFER_SIZE`.